app/src/components/user_details_form.rs

```diff
@@ -413,6 +418,8 @@ impl UserDetailsForm {
         if let Some(avatar) = &self.avatar {
             user_input.avatar = Some(to_base64(avatar)?);
         }
+        user_input.removeAttributes = remove_attributes;
```

Comment:
That might very well be what I was hitting last time I was trying to fix this code. You're the 3rd author of this chain of WIP PRs, btw :)